.github/workflows/publish-c-apidocs.yml

```diff
@@ -21,27 +27,17 @@ jobs:
           sudo apt-get install libclang-cpp14
           wget https://www.doxygen.nl/files/doxygen-1.9.6.linux.bin.tar.gz
           tar xvzf doxygen-1.9.6.linux.bin.tar.gz
-      - name: Set commit ID
-        id: vars
-        run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
       - name: Run doxygen
         run: |
           mkdir -p build/doxygen
           cd docs/c_cxx
           ../../doxygen-1.9.6/bin/doxygen
-      - uses: actions/checkout@v2
-        with:
-          ref: gh-pages
-          clean: false
-      - name: Move API docs into target area
-        run: |
-          rm -rf docs/api/c
-          mv build/doxygen/html docs/api/c
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v3
+      - name: Log source commit
+        run: git rev-parse --short HEAD > build/doxygen/html/source-version.txt
```

Comment:
so will the users only be able to see the git commit by going to a page like this one?
https://natke.github.io/onnxruntime/docs/api/c/source-version.txt

might be nice if it was visible from the other API doc pages as well. this is a good start though.

Reply:
Yes that is how you would see it. Let me know any other suggestions you have for this. Happy to implement. It will likely be rarely accessed but nice to be able to audit it

Reply:
if something on the actual API docs pages could say what ORT version and commit the docs were generated from that would be nice. e.g., the Java docs already show the ORT version.

Reply:
This should not be too hard to do. I will add a workitem for it